temp/README.md

```diff
@@ -0,0 +1,92 @@
+# angular-cli-esri-map
```

Comment:
Hi, not sure what the `\temp` directory is for?

Reply:
deleted - was using this to stage the original files as I was adding them to the cli project but should have been removed before I opened the PR